src/WordPress/PluginActions.php

```diff
@@ -53,6 +53,9 @@ class PluginActions extends AbstractPluginActions
         'integrationName',
     ];
 
+    const USER_CONFIG_PATH = '/../../config.json';
```

Comment:
I remember at some point there was a script that nuked the root level config and composer jsons. Can we confirm that these exist in the release zips?

Reply:
yep, we definitely do keep it around. both archives (GitHub + WordPress SVN) are showing it as present.

```
$ wget https://downloads.wordpress.org/plugin/cloudflare.4.2.1.zip
$ unzip -l cloudflare.4.2.1.zip | grep -e "composer\.json" -e "config\.json"
     1309  03-05-2021 20:13   cloudflare/composer.json
      807  03-05-2021 20:13   cloudflare/config.json
      879  12-12-2017 15:07   cloudflare/vendor/cloudflare/cf-ip-rewrite/composer.json
      594  10-02-2020 04:01   cloudflare/vendor/cloudflare/cloudflare-plugin-backend/composer.json
      561  06-29-2020 13:39   cloudflare/vendor/psr/log/composer.json
```